cmssw-tool-conf.spec

```diff
@@ -119,6 +119,7 @@ Requires: fftjet-toolfile
 Requires: lapack-toolfile
 Requires: pyminuit2-toolfile
 Requires: professor-toolfile
+Requires: professor2-toolfile
```

Comment:
@mkirsano -please summarize the request here (and its expected longevity) - its certainly not normal to try to manage multiple versions of the "same" tool around.